src/Kusto/Kusto/ChangeLog.md

```diff
@@ -18,6 +18,12 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Capacity is a new and optional parameter for Create and Update Cluster.
+* ETag of Database is depricated.
+* Cluster has new Properties: Uri, DataIngestionUri and Capacity.
+
```

Comment:
There should only be one extra new line between the bottom line of your description and the next version.